ui/styles/src/base/_admin.scss

```diff
@@ -486,8 +479,9 @@ div.pods-manage-field .pods-field-option-group .pods-field-option-group-label {
 /**
  * Pods DFV
  */
-.pods-manage-field .pods-dfv-field .pods-pick-values > ul {
-    border: 0;
+.pods-manage-field .pods-dfv-field .pods-ui-list-autocomplete {
```

Comment:
The images view should be similar right?
`.pods-manage-field .pods-dfv-field .pods-ui-file-list.pods-field-template-rows`.